HW_2.1.js

```diff
@@ -0,0 +1,32 @@
+function durationBetweenDates(startDate, endDate, unit) {
+    unit = unit || "emptyValue";
```

Comment:
```suggestion
function durationBetweenDates(startDate, endDate, unit = 'days') {
```

Значення за замовчуванням задаються в параметрах функції, як ми проходили на вебінарах, та мають в собі містити якісь значення які функція може використати в своїй логіці. `emptyValue` нам нічого не дасть. Приклад вище.

`startDate` та `endDate` також повинні мати значення за замовчуванням. Бо default у свічі не обчислюється, ви там фактично повертаєте текст помилки.

Загалом треба орієнтуватися на текст завдання, якщо там вказано "Також вкажіть значення по замовчуванню для всіх цих параметрів (на ваш вибір)." значіть має бути відповідна реалізація :)